.github/workflows/deploy-to-environment.yml

```
@@ -90,3 +90,4 @@ jobs:
         RESOURCE_WHITELIST: ${{ secrets.RESOURCE_WHITELIST }}
         MASKINPORTEN_ENVIRONMENT: ${{ secrets.MASKINPORTEN_ENVIRONMENT }}
         APIM_IP: ${{ secrets.APIM_IP }}
+        CORRESPONDENCE_DB_PASSWORD: ${{ secrets.DB_PWD }}
```

actionlint (1.7.4)

input "CORRESPONDENCE_DB_PASSWORD" is not defined in action "Deploy to Environment" defined at "./.github/actions/deploy-to-environment". available inputs are "ACCESS_MANAGEMENT_SUBSCRIPTION_KEY", "APIM_IP", "AZURE_CLIENT_ID", "AZURE_ENVIRONMENT_KEY_VAULT_NAME", "AZURE_NAME_PREFIX", "AZURE_STORAGE_ACCOUNT_NAME", "AZURE_SUBSCRIPTION_ID", "AZURE_TENANT_ID", "AZURE_TEST_ACCESS_CLIENT_ID", "CONTACT_RESERVATION_REGISTRY_BASE_URL", "CORRESPONDENCE_BASE_URL", "DB_PWD", "DIALOGPORTEN_ISSUER", "IDPORTEN_CLIENT_ID", "IDPORTEN_CLIENT_SECRET", "IDPORTEN_ISSUER", "MASKINPORTEN_CLIENT_ID", "MASKINPORTEN_ENVIRONMENT", "MASKINPORTEN_JWK", "MASKINPORTEN_TOKEN_EXCHANGE_ENVIRONMENT", "PLATFORM_BASE_URL", "PLATFORM_SUBSCRIPTION_KEY", "RESOURCE_WHITELIST", "SBL_BRIDGE_BASE_URL", "SLACK_URL", "environment", "imageTag"

(action)